<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorHighlighting</class>
 <widget class="QDialog" name="ColorHighlighting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>253</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="example2_label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>K1ABC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="example1_label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>K1ABC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="p1_label">
     <property name="text">
      <string>CQ in message</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="example3_label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>K1ABC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="p2_label">
     <property name="text">
      <string>My Call in message</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="p3_label">
     <property name="text">
      <string>Transmitted message</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="example4_label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>K1ABC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="p4_label">
     <property name="text">
      <string>New DXCC</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="p6_label">
     <property name="text">
      <string>New Grid</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="p5_label">
     <property name="text">
      <string>New DXCC on Band</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="example5_label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>K1ABC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="example6_label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>K1ABC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="example7_label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>K1ABC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="p8_label">
     <property name="text">
      <string>New Call</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="p7_label">
     <property name="text">
      <string>New Grid on Band</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="example8_label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>K1ABC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="example9_label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>K1ABC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="example10_label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>K1ABC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="p9_label">
     <property name="text">
      <string>New Call on Band</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="p10_label">
     <property name="text">
      <string>Uploads to LotW</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="example11_label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>K1ABC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="example12_label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>K1ABC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="example13_label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>K1ABC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="example14_label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>K1ABC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="example15_label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>K1ABC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="example16_label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>K1ABC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="p11_label">
     <property name="text">
      <string>New Continent</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="p12_label">
     <property name="text">
      <string>New Continent on Band</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="p13_label">
     <property name="text">
      <string>New CQ Zone</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="p14_label">
     <property name="text">
      <string>New CQ Zone on Band</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="p15_label">
     <property name="text">
      <string>New ITU Zone</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="p16_label">
     <property name="text">
      <string>New ITU Zone on Band</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
